### PR TITLE
Add environment variables to syscheck documentation

### PR DESCRIPTION
**Pull request by antoniomanuelfr**
## Description
Hi team.

As mentioned in #2433, syscheck now supports Linux environment variables and this new feature needs to be added to the documentation.

Kind regards.

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns.

### CODE REVIEW

**Review by bah07 — APPROVED** [review]
LGTM!

File: source/user-manual/capabilities/file-integrity/fim-configuration.rst [hunk 1]
```diff
@@ -40,6 +40,30 @@ To configure syscheck, a list of files and directories must be identified. The `
...
+
+Environment variables can be used to configure syscheck in Linux and Windows.
+
+.. code-block:: xml
+
+  <syscheck>
+    <directories check_all="yes">$DIRECTORY</directories>
+  </syscheck>
+
+On UNIX based systems, the variable must be added to the file ``/etc/ossec-init.conf`` if wazuh is restarted using systemd. On the other hand, if wazuh is restarted using the ``ossec-control`` binary, the variable must be owned by the root user.
```
Comment (davidfmiranda) [inline source/user-manual/capabilities/file-integrity/fim-configuration.rst:53]:
```suggestion
On UNIX based systems, the variable must be added to the file ``/etc/ossec-init.conf`` if Wazuh is restarted using systemd. On the other hand, if Wazuh is restarted using the ``ossec-control`` binary, the variable must be owned by the root user.
```

**Review by davidfmiranda — CHANGES_REQUESTED** [review]
Hello @antoniomanuelfr, I've reviewed your PR and LGTM, nevertheless, I've suggested you two capitalize `Wazuh`

Regards,

David

**Review by bah07 — APPROVED** [review]

**Review by davidfmiranda — APPROVED** [review]
LGTM

**Pull request merged**